internal/cmd/internal/olmv1/alpha_install.go

```diff
@@ -0,0 +1,30 @@
+package olmv1
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/kubectl-operator/internal/cmd/internal/log"
+	experimentalaction "github.com/operator-framework/kubectl-operator/internal/pkg/experimental/action"
+	"github.com/operator-framework/kubectl-operator/pkg/action"
+)
+
+func NewOperatorInstallCmd(cfg *action.Configuration) *cobra.Command {
+	i := experimentalaction.NewOperatorInstall(cfg)
+	i.Logf = log.Printf
```

Comment:
When you set `i.Logf` here does it purely set the log method for the `OperatorInstall` action or could you also use it in this file on lines 23 + 25 i.e. `log.Fatalf(...)` becomes `i.Logf(...)`? (mostly a knowledge question rather than an actual change request)

Reply:
_Disclaimer:_ this was based on Joe's code; I pulled it in with minimal changes.

AFAICT, setting `i.Logf` is to set the logging destination for the action that is about to happen (i.e. "Logf" = "log function"). It's not really meant to be called as a general logging function. It's clearer to keep lines 23 and 25 as "log.Somethingf", rather than hide it behind `i.Logf`. golang discourages accessors, but this would be a classic case of a set accessor for specify log output.